tide/Cargo.toml

```diff
@@ -19,34 +19,27 @@ repository = "https://github.com/rustasync/tide"
 version = "0.2.0"
 
 [dependencies]
-cookie = { version = "0.12", features = ["percent-encode"] }
 futures-preview = "0.3.0-alpha.16"
-fnv = "1.0.6"
 http = "0.1"
 http-service = "0.2.0"
-pin-utils = "0.1.0-alpha.4"
-route-recognizer = "0.1.12"
 serde = "1.0.91"
 serde_derive = "1.0.91"
-serde_json = "1.0.39"
 slog = "2.4.1"
 slog-async = "2.3.0"
 slog-term = "2.4.0"
-typemap = "0.3.3"
 serde_urlencoded = "0.5.5"
-
-[dependencies.http-service-hyper]
-optional = true
-version = "0.2.0"
+tide-cookies = { path = "../tide-cookies", optional = true }
+tide-core = { path = "../tide-core", optional = true }
```

Comment:
Should this be optional? It looks like it's only included in the `hyper` feature so building with `--no-default-features` probably doesn't work.

Reply:
Yep, just tested locally and this fails with

```console
> cargo build --no-default-features --manifest-path tide/Cargo.toml --verbose
```

CI will need some work to be able to test feature flags accurately, I'll open an issue and probably try and port some of the stuff we have setup in `futures` here for it.

Reply:
Good catch! Fixed.